### PR TITLE
build(git-hooks): Change to always suggest `make` when dependencies are out of date

### PR DESCRIPTION
**Pull request by billyvg**
This changes the post-merge git hook to allow users to copy a single command to update their dependencies.
Also fixes check for migrations (was previously checking for
`south_migrations`)

image

### CODE REVIEW

File: config/hooks/post-merge [hunk 1]
```diff
@@ -9,26 +9,38 @@ trap "rm -f ${files_changed_upstream}" EXIT
...
+[[ "$py" || "$js" || "$migrations" ]] && cat <<EOF
+
+${red}Run ${bold}${update_command}${reset}${red}to update necessary dependencies${reset}
+
 EOF
 
-echo
+
+echo "Do you want to run updates now? [y/n] "
+read answer
```
Comment (dashed) [inline config/hooks/post-merge]: If this is going to be the default, can there be an env var to skip this?

**Review by joshuarli — CHANGES_REQUESTED** [review]
I feel like the bullet points are unnecessary and confusing (a python venv isn't a dependency), could we just keep the original text? Like "It looks like some dependencies have changed that will require your intervention." and then the "Run make ..." stuff.

File: config/hooks/post-merge (hunk 1 above)
Comment (joshuarli) [reply config/hooks/post-merge]: Looks like we've decided to not prompt after all as it'd be a potential headache with editor/ides.

Comment (dashed) [reply config/hooks/post-merge]: Ok.

**Review by joshuarli — APPROVED** [review]
Would be worth letting people know about setting `SENTRY_POST_MERGE_AUTO_UPDATE` to something if they want to autoupdate, if they don't have it set. Unless you want to keep this feature hidden.

**Pull request merged**